test/data/StorageFileTest/ValidData.txt

```diff
@@ -1,2 +1,2 @@
-John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01
-Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/friend t/criminal
+John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01 s/ 
+Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/friend t/criminal s/ 
```

Comment:
Is the star tag a compulsory field?

Reply:
The star field is not compulsory during the adding of contacts. But will be displayed when any list command is called.